src/components/pages/Categories/List.vue

```diff
@@ -53,13 +58,27 @@ export default {
     errorMessage() {
       return this.$store.state.categories.errorMessage;
     },
+    deleteCategoryName() {
+      return this.$store.state.categories.deleteCategory.name;
+    },
   },
   created() {
     this.$store.dispatch('categories/getAllCategories');
     this.$store.dispatch('categories/initCategory');
     this.$store.dispatch('categories/clearMessage');
   },
   methods: {
+    openModal(categoryId, categoryName) {
+      const categoryData = { id: categoryId, name: categoryName };
+      this.$store.dispatch('categories/confirmDeleteCategory', categoryData);
+      this.toggleModal();
+    },
+    handleClick() {
+      this.$store.dispatch('categories/deleteCategory').then(() => {
+        this.$store.dispatch('categories/getAllCategories');
+      });
+      this.toggleModal();
```

Comment:
[imo]

非同期処理になっていることで削除しながらモーダルも閉じられるというような動作になってしまっているので、以下のような記述にするのが良いかなと思います。

```suggestion
      this.$store.dispatch('categories/deleteCategory').then(() => {
        this.$store.dispatch('categories/getAllCategories');
        this.toggleModal();
      });
```

Reply:
854f73b0baec394a884dc5d7b34f5ee9151b255e
こちら、上記のように修正済みです！！